en/configure-a-tidb-cluster.md

```diff
@@ -557,6 +557,20 @@ The YAML file above:
 
 When Kubernetes deletes the TiDB Pod, it also removes the TiDB node from the service endpoints. This is to ensure that the new connection is not established to this TiDB node. However, because this process is asynchronous, you can make the system sleep for a few seconds before you send the `kill` signal, which makes sure that the TiDB node is removed from the endpoints.
 
+## Configure graceful upgrade for TiKV cluster
+
+During TiKV upgrade, TiKV Pod is restarted only after the Operator evicts all Region leaders on TiKV Pod (which means the Region leader number drops to 0) or the eviction reaches the specified timeout (10 minutes by default).
+
+If you want to specified the timeout, you can specified `spec.tikv.evictLeaderTimeout` (10 minutes by default).
+
+If you do not want it to restart after timeout with having not evicting all leaders, you can just specified a large value like:
```

Comment:
```suggestion
If the eviction of Region leaders exceeds the specified time-out, restarting TiKV Pod causes issues such as failures of some requests or more latency. To avoid the issues, you can configure the timeout `spec.tikv.evictLeaderTimeout` (10 minutes by default) to a larger value. For example:
```